libs/domain/picking/src/components/filters/filters.component.ts

```diff
@@ -72,6 +77,13 @@ export class FiltersComponent extends LitElement {
     this.form.requestSubmit();
   }
 
+  protected onFormKeyDown(event: KeyboardEvent): void {
+    if (event.key === 'Enter') {
+      event.preventDefault();
+      this.onApply();
```

Comment:
Please move `if (event.key === 'Enter') {event.preventDefault()} ` to apply method